src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
```

Comment:
Consider using a constant for the daily rental cost instead of hardcoding the value '40'. This will make the code more maintainable and self-explanatory.